tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -3248,6 +3248,58 @@ TEST_P(OpConverter_FP32_FP16_Test, ConvertSquare) {
                   ArrayFloatNear(expected_outputs, 0));
 }
 
+TEST_P(OpConverter_FP32_Test, ConvertFill) {
+  Scope s = Scope::NewRootScope();
+  auto dims = ops::Placeholder(s.WithOpName("dims"), DT_INT32);
+  auto value = ops::Placeholder(s.WithOpName("value"), tf_type_);
+  auto fill = ops::Fill(s.WithOpName("my_fill"), dims, value);
+  const NodeDef& node_def = fill.operation.node()->def();
+
+  if (trt_mode_ == TrtTestMode::kImplicitBatch) {
+    Reset();
+    // random data
+    AddTestWeights("dims", {2}, {2,2}, DT_INT32);
+    AddTestWeights("value", {1}, {42.0}, tf_type_);
+    RunValidationAndConversion(node_def, error::UNIMPLEMENTED,
+                               "Conversion for Fill is not implemented in"
+                               "implicit batch mode");
+    return;
+  }
+
+  std::vector<std::vector<int>> test_dims = {
+    {8},
+    {8, 2, 4},
+    {32, 32, 3200}};
+
+  float val = 42.0;
+  Status status = Status::OK();
+  for (bool dims_is_tensor : {true, false}) {
+    for (bool value_is_tensor : {true, false}) {
+      for (auto input_dims : test_dims) {
+        Reset();
+        std::vector<int32> dims_dims = {input_dims.size()};
+        if (dims_is_tensor) {
+          AddTestTensor("dims", dims_dims, DT_INT32, input_dims, dims_dims);
+        } else {
+          AddTestWeights("dims", dims_dims, input_dims, DT_INT32);
+        }
+        if (value_is_tensor) {
+          AddTestTensor("value", {1}, tf_type_, {val});
```

Comment:
You test with 1 element vector only. Please check also for a value tensor that is a scalar (shape = {}).